pkg/arvo/sur/group-store.hoon

```diff
@@ -1,19 +1,37 @@
+/-  *group
 |%
-+$  group  (set ship)
+::  $action: request to change group-store state
 ::
-+$  group-action
-  $%  [%add members=group pax=path]                :: add member to group
-      [%remove members=group pax=path]             :: remove member from group
-      [%bundle pax=path]                           :: create group at path
-      [%unbundle pax=path]                         :: delete group at path
-  ==
+::    %add-group: add a group
+::    %add-members: add members to a group
+::    %remove-members: remove members from a group
+::    %add-tag: add a tag to a set of ships, creating the tag if it doesn't exist
+::    %remove-tag:
+::      remove a tag from a set of ships. If the set is empty remove the tag
+::      from the group.
```

Comment:
What does it mean to "create" a tag? We don't track them explicitly as their own set of possibilities (only implicitly as the set of tags that are actively in use), so feels weird to talk about them that way.